src/dashboard/src/settings/local.py

```diff
@@ -30,6 +30,12 @@
 DEBUG = True
 TEMPLATES[0]["OPTIONS"]["debug"] = True
 
+# Use Django uncompressed StaticFilesStorage rather than compressed Whitenoise
+# storage in local development so that tests run correctly.
```

Comment:
>so that tests run correctly

Since the intent is to make tests work, I wouldn't switch the `local` settings but the `tests` ones. That brings [a different problem](https://github.com/archivematica/Issues/issues/1218#issuecomment-637034567) unfortunately.

AFAIU `Whitenoise` works fine for development, but let me know if I'm missing something.